godtools/App/Features/Tools/SwiftUI/AllToolsContentView.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  AllToolsContentView.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 4/6/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import SwiftUI
+
+struct AllToolsContentView: View {
```

Comment:
Yeah, I've noticed some weird stuff like that, I think because of the way I'm removing cell separators right now (each iOS version seems to require a different way ☹️).  I have a TODO to go back and iron that out in https://jira.cru.org/browse/GT-1528, and I went ahead and noted what you saw in that ticket as well.

Reply:
Shoot those UI bugs can be difficult to pin down sometimes too.  Maybe do some stackoverflow searching too and see if anyone else has reported a similar bug.